gradle.properties

```diff
@@ -4,18 +4,18 @@ org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 	# check these on https://fabricmc.net/versions.html
 	minecraft_base_version=1.19
```

Comment:
Thanks for the PR, looks great! My only nitpick is that the MC base version must be bumped to 1.19.3, since this is absolutely not a backwards-compatible port
Cheers